core/src/main/java/org/apache/carbondata/core/carbon/datastore/chunk/impl/FixedLengthDimensionDataChunk.java

```diff
@@ -77,13 +79,75 @@ public FixedLengthDimensionDataChunk(byte[] dataChunk, DimensionChunkAttributes
       rowId = chunkAttributes.getInvertedIndexesReverse()[rowId];
     }
     int start = rowId * chunkAttributes.getColumnValueSize();
+    int dict = getInt(chunkAttributes.getColumnValueSize(), start);
+    row[columnIndex] = dict;
+    return columnIndex + 1;
+  }
+
+  @Override public int fillConvertedChunkData(ColumnVectorInfo[] vectorInfo, int column,
+      KeyStructureInfo restructuringInfo) {
```

Comment:
Here is the interface implementation method, this class does not use but other implementation class like `ColumnGroupDimensionDataChunk` uses it.